docs/success-guide/setup_config_files.md

```diff
@@ -126,25 +126,46 @@ Limits on deletion prevent accidental account deletion in the event of misconfig
 
 
 
-#### Delete Protection*
+#### Delete Protection
 
 If you want to drive account creation and removal through User Sync, and want to manually create a few accounts then you may need this feature to keep User Sync from deleting your manually created accounts.
 
-&#9744; If you need to use this feature, add the lines such as below to the config file at the top level.  Note that this feature is not available at this point in time.
+&#9744; If you need to use this feature, add lines such as below to the config file at the top level.   You can create user groups and add users to that user group on the Adobe Admin Console, and then list that user group as excluded from User Sync processing to protect that set of users.  You can list specific users and/or a pattern that matches specific user names to protect those users.  You can protect users based on their identity type as well.  For example, often User Sync is used only to manage federatedID or enterpriseID user types and you can exclude adobeID type users from management by User Sync.  You only need to include configuration items for exclusions that you wish to use.
 
-	user_removal:
-	    - exclude_group: special_users   
-	    - exclude:   ".*@example.com"   
-	    - exclude_adobe_id              
+```YAML
+dashboard_user_exclusions:
+  exclude_groups: 
+    - administrators   # Names an Adobe user group or product configuration whose members are not to be altered or removed by User Sync
+    - contractors      # You can have more than one group in a list
+  exclude_users:
+    - ".*@example.com"
+    - important_user@gmail.com
+  exclude_identity_types:
+    - adobeID          # adobeID, enterpriseID, and/or federatedID
+```
+
+
+In the above, administrators, contractors, and the user names are example values.  You would use the names of Adobe user groups, product configurations, or users that you have created.
+
+`exclude_groups` defines a list of Adobe user groups or product configurations. Adobe users who are members of such groups are neither removed, updated, nor have their group membership changed.
```

Comment:
- change to "list of Adobe user groups, product configurations, or both"
- change "such groups" to "listed groups"